plotting_refactor/Dataset.py

```diff
@@ -0,0 +1,63 @@
+import time
+
+
+class Dataset:
```

Comment:
I get that a toy model dataset might not need lots of comments, but maybe one docstring to say what it is